middleware.js

```diff
@@ -265,18 +269,18 @@ module.exports = function(options) {
 
     // Compare mtimes
     fs.stat(sassPath, function(err, sassStats) {
-      if (err) {
-        error(err);
+      if (err) { // sassPath can't be accessed, nothing to compile
         return next();
       }
 
       fs.stat(cssPath, function(err, cssStats) {
-        if (err) { // CSS has not been compiled, compile it!
-          if ('ENOENT' === err.code) {
-            if (debug) { log('error', 'not found', cssPath); }
+        if (err) {
+          if ('ENOENT' === err.code) { // CSS has not been compiled, compile it!
+            if (debug) { log('debug', 'not found', cssPath); }
```

Comment:
not finding the CSS file is common, not really an error - it happens before the first compilation